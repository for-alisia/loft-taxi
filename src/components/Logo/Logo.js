import React from 'react';
import logoImg from './logo.png';
import './Logo.css';

const Logo = () => {
    return (
        <div className='Logo'>
            <img className='Logo__img' src={logoImg} alt='Logo Taxi' />
        </div>
    );
};

export default Logo;
