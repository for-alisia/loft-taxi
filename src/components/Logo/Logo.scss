.Logo {
    width: 50%;
    text-align: center;
    animation: moveToRight 0.5s ease;
}

@keyframes moveToRight {
    0% {
        transform: translateX(-60rem);
        opacity: 0;
    }
    100% {
        transform: translateX(0);
        opacity: 100%;
    }
}
