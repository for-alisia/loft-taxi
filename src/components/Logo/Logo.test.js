import React from 'react';
import Logo from './Logo';
import { shallow } from 'enzyme';

describe('Logo component', () => {
    it('renders logo customs', () => {
        expect(shallow(<Logo />)).toMatchSnapshot();
    });
});
