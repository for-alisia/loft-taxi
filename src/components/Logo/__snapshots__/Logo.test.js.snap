// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component renders logo customs 1`] = `ShallowWrapper {}`;
